lib/xboxrt/libc_extensions/stdlib_ext_.h

```diff
@@ -32,6 +32,10 @@ int mbtowc (wchar_t *pwc, const char *string, size_t n);
 #define __min(a,b) (((a) < (b)) ? (a) : (b))
 #define __max(a,b) (((a) > (b)) ? (a) : (b))
 
+#define _byteswap_ushort __builtin_bswap16
+#define _byteswap_ulong __builtin_bswap32
+#define _byteswap_uint64 __builtin_bswap64
```

Comment:
#433 was merged recently, so the win32 byteswap functions no longer have to be a part of this.